protobuf/cluster/database.proto

```diff
@@ -23,20 +23,27 @@ option java_outer_classname = "DatabaseProto";
 package grakn.protocol.cluster;
 
 message Database {
-    message Discover {
+
+    message Replica {
+        string address = 1;
+        string database = 2;
+        Rank rank = 3;
+        int64 term = 4;
+
+        enum Rank {
+            PRIMARY = 0;
+            SECONDARY = 1;
+            BACKUP = 2;
+        }
```

Comment:
The three ranks are:

- `PRIMARY`: This replica should be contacted to initiate primary sessions
- `SECONDARY`: This replica should be contacted to initiate secondary sessions
- `BACKUP`: This replica should only be contacted if a primary or secondary replica fails